plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/TestDeltaLakeMetastoreAccessOperations.java

```diff
@@ -278,6 +283,143 @@ public void testDropTable()
                         .build());
     }
 
+    @Test
+    public void testInformationSchemaColumns()
```

Comment:
I'm not sure whether this test is needed in the context of this PR because we're interested here to minimize the object storage calls, not the metastore calls.

Reply:
I wanted to verify `REPLACE_TABLE` count just in case, but yes it's not mandatory. Reverted this change and updated `TestDeltaLakeFileOperations`.